### ISSUE TITLE
Scaling the loss to be able to use 32bit float again

### ISSUE THREAD

**Issue by FrederikLauber** [keras-team/keras#7984]
Hello,

is there a way to rescale the loss in keras?
I have a lot of samples (10E6) and even in epoch 1, i am already at an loss of E-6.
Can I scale this up so that I can again use 32bit precision?
Thanks in advance

**Comment by harshini-gadige** [comment]
Hi

This issue isn't related to a bug/enhancement/feature-request or other accepted types of issue.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please open another issue.
